test/onnx_import.cpp

```diff
@@ -1536,6 +1539,74 @@ TEST(onnx, model_matmul_vec_ten3d)
     EXPECT_TRUE(test::all_close_f(expected_output.front(), outputs.front()));
 }
 
+TEST(onnx, model_softplus)
+{
+    auto function =
+        onnx_import::import_onnx_model(file_util::path_join(SERIALIZED_ZOO, "onnx/softplus.onnx"));
+
+    // -1.0f, 0, 1.0f, 10.f,                    normal input values for activation
+    // 100.0f, -100.0f, 1000.0f, -1000.0f,      input values that leads to exp() overflow
+    // FLT_MIN, FLT_MIN / 10, -FLT_MIN / 10,    min, denorm, -denorm
```

Comment:
Comment not in sync with values you updated below. At request from @diyessi you updated to / 16. Need to update that here too.